js/views/modals/editListing/VariantInventoryItem.js

```diff
@@ -82,11 +82,15 @@ export default class extends BaseView {
   }
 
   calculateTotalPrice(surcharge) {
-    const updatedSurcharge = (typeof surcharge === 'number' && !isNaN(surcharge)) ?
-      surcharge : 0;
+    const listingPrice = this.options.getPrice();
```

Comment:
This is also an additional fix where the variant inventory table was bombing if the listing price was not a number. Now it just won't display the total price if it's unable to calculate it (if either the listing price or surcharge are not numbers). If the user saves the listing the offending field will get an error instructing the user to provide a proper number.